DESCRIPTION

```diff
@@ -3,7 +3,8 @@ Type: Package
 Title: SDTM Data Transformation Engine
 Version: 0.1.0
 Authors@R: c(
-    person("Rammprasad", "Ganapathy", role = c("aut", "cre"),
+    person("Rammprasad", "Ganapathy", email = "ganapathy.rammprasad@gene.com",
+    role = c("aut", "cre"),
```

Comment:
Was this one one the comments? It seems pipeline is failing and I wonder whether this is causing it.

Reply:
Please see https://github.com/pharmaverse/sdtm.oak/actions/runs/10605052955/job/29393055019?pr=94